portfolio/src/main/webapp/WEB-INF/locations.csv

```diff
@@ -0,0 +1,3 @@
+Pittsburgh,My home city,40.439709,-79.9959
+Philadelphia,UPENN,39.952145,-75.193074
+Preikestolen,Cliff in Norway,58.986495,6.188502
```

Comment:
Since you weren't taught how to make test cases yet, I would add some incorrectly formatted locations and see if ur website break

Reply:
How much error catching should i do here? I didn't think the user would be able to modify the csv file, and I would be able to fix any errors before deploying the website but i'm not sure if this is a good assumption.